src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4543,53 +4545,94 @@ private StatementAst EnumDefinitionRule(List<AttributeBaseAst> customAttributes,
                 return new ErrorStatementAst(enumToken.Extent);
             }
 
-            SkipNewlines();
-            Token lCurly = NextToken();
-            if (lCurly.Kind != TokenKind.LCurly)
+            TypeConstraintAst underlyingTypeConstraint = null;
+            var oldTokenizerMode = _tokenizer.Mode;
+            try
             {
-                // ErrorRecovery: If there is no opening curly, assume it hasn't been entered yet and don't consume anything.
+                SetTokenizerMode(TokenizerMode.Signature);
+                Token colonToken = PeekToken();
+                if (colonToken.Kind == TokenKind.Colon)
+                {
+                    this.SkipToken();
+                    SkipNewlines();
+                    ITypeName underlyingType;
+                    Token unused;
+                    underlyingType = this.TypeNameRule(allowAssemblyQualifiedNames: false, firstTypeNameToken: out unused);
+                    if (underlyingType == null)
+                    {
+                        ReportIncompleteInput(After(colonToken),
+                            nameof(ParserStrings.TypeNameExpected),
+                            ParserStrings.TypeNameExpected);
```

Comment:
Please align all arguments to one column.
Below too (there is many cases).

Reply:
Fixed all instances of `ReportError()` and `ReportIncompleteInput()`